Makefile.toml

```diff
@@ -127,6 +127,7 @@ args = [
 [tasks.build-contract]
 category = "Build"
 command = "${CARGO}"
+toolchain = "1.69"
```

Comment:
@olga24912 could you please clarify why? There is a `1.74.1` in the rust-toolchain.